### DISCUSSION TITLE
Wachtwoord vergeten knop in https://demo.sensorenregister.nl/

### DISCUSSION THREAD

**Discussion by CelineJansen** [kadaster-labs/sensrnet-home#192] [Q&A] [answered]
De wachtwoord vergeten knop in https://demo.sensorenregister.nl/ werkt niet, ik krijg de volgende response (vorige week was dit ook al zo):
![image](https://user-images.githubusercontent.com/79267472/121329840-d6ca2400-c915-11eb-901e-31a99244191a.png)

**Comment by kad-busses** [comment, marked as answer]
Hoi Celine. Bedankt voor het melden, dit was nog niet bij ons bekend. Ik ben er net in gedoken en zie dat dit nog ontbrekende functionaliteit is, wat natuurlijk erg onhandig is. Ik kan volgende week kijken of ik dat alsnog toe kan voegen. We zijn binnen het Kadaster overigens sowieso bezig een alternatief gebruikerssysteem voor de demo-omgeving te onderzoeken, waarin dit netter geregeld is. 

Wat ik voor nu voor je kan doen, is om je wachtwoord handmatig te resetten. Je krijgt dan een eenmalige code van mij waarmee je kunt inloggen, waarna je direct je wachtwoord kunt wijzigen. Dan kun je weer gewoon gebruik maken van de testomgeving. Is dat oké? Dan zal ik je een mail sturen.

**Reply by CelineJansen** [reply]
Jij bedankt voor de snelle reactie! Een handmatige reset zou super zijn (en dan ga ik vanaf nu zorgvuldiger om met het opslaan van dit wachtwoord, zodat je niet aan de gang blijft met handmatig moeten resetten ;) ).
